src/game/game_gui.cpp

```diff
@@ -367,12 +367,14 @@ struct GSConfigWindow : public Window {
 
 	void OnDropdownSelect(int widget, int index) override
 	{
+		if (widget >= 0) return;
```

Comment:
Why not make a sentinel `WID_GSC_DROPDOWN = -1` or something similar in widgets/game_widgets.h. Then use that here and in the calls to create the drop downs? Seems better than the implicit assumption that the widget for the callback is always going to be some negative value for those cases.